packages/store-indexer/src/sqlite/apiRoutes.ts

```diff
@@ -44,5 +45,44 @@ export function apiRoutes(database: BaseSQLiteDatabase<"sync", any>): Middleware
     }
   });
 
+  router.post("/api/sqlite-indexer", async (ctx) => {
```

Comment:
should we name this `/q` or whatever the dozer one is?

(this is the sqlite indexer so we don't need to name the route that too)

Reply:
also I wonder if we should put this behind some sort of flag so folks can opt-in because this is exposing the entire DB to any arbitrary SQL, and not sure the indexer API has done this before

Reply:
It would be convenient if it was `/q` although not sure if it's the best naming tbh, wdyt? Thinking of either `/q`, `/api/q` or `/api/data`.

As for arbitrary SQL, yeah, it's safe for development but if the endpoint is exposed in anyway, not ideal. What about if we limit it to only `SELECT` statements? And still can add a flag.

Reply:
I'm not confident we can reliably prevent all kinds of attacks, so I'd rather this endpoint be an opt-in flag

if we're aiming to create something dozer-like I'd prob just follow dozer's naming for now (can't recall if it's `/q` or `/api/q`)